README.md

```diff
@@ -160,7 +163,7 @@ We :heart: your contribution. The [community](https://github.com/kubesphere/comm
 
 Please submit any KubeSphere bugs, issues, and feature requests to [KubeSphere GitHub Issue](https://github.com/kubesphere/kubesphere/issues).
 
-## Who are using KubeSphere
+## Who are Using KubeSphere
```

Comment:
Why do we capitalize `using`?

Reply:
I updated all headings to the sentence case according to the [Google Doc Style Guide](https://developers.google.com/style/capitalization).